datasets/ONS-UK-trade-in-services-by-business-characteristics/info.json

```diff
@@ -15,11 +15,45 @@
     },
     "transform": {
         "airtable": "recvEuJKTaVY6Ft4n",
+        "columns": {
+            "Period": {
+                "source": "",
+                "parent": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+                "value": "http://reference.data.gov.uk/id/{+period}"
+            },
+            "Business size": {
+                "parent": "http://gss-data.org.uk/def/family-trade/concept-scheme/employment-size-bands",
```

Comment:
The `parent` key here defines an `rdfs:subPropertyOf` relationship between the dimension and the `parent`. Throughout you specify `/concept-scheme/` resources to be parents, meaning you're saying that the dimension is a `subPropertyOf` a concept scheme, which may not make complete sense!